### ISSUE TITLE
Automatically assign icons once icon library has been added

### ISSUE THREAD

**Issue by Alexander-Wilms** [beemdevelopment/Aegis#1074]
After one has added an icon library one still has to set the icon for each entry. Instead, Aegis could automatically assign the first recommended icon for each entry and for each new entry as well.

**Comment by alexbakker** [comment]
Duplicate of #773.

**Comment by beyond9thousand** [comment]
Sorry to be bumping this again, but as an end user I'm a little curious if you don't mind the question. Other than the task not being high priority, and the fact that I should just be creating my own patch if I'm impatient - is there a huge technical obstacle preventing the implementation of this feature? In layman terms, if possible. Genuinely asking

**Comment by alexbakker** [comment]
@beyond9thousand No technical obstacles. It's purely a matter of someone taking the time to implement this.
